css-scroll-anchoring-1/Overview.bs

```diff
@@ -85,16 +86,41 @@ An anchor node can be any <a>box</a> except one for a non-<a>atomic inline</a>.
 The anchor node is always a <a>descendant</a> of the <a>scrolling box</a>.
 In some cases, a scrolling box may not select any anchor node.
 
+An element |C| is a <dfn id="anchor-viable-candidate">viable candidate</dfn>
+for becoming a scroll anchor for a scrolling box |S| if it meets all of the
+following criteria:
+  * |C| is a non-<a>atomic inline</a>.
+  * |C| is <a>partially visible</a> or <a>fully visible</a> in |S|
+  * |C| is a descendant of |S|
+  * |C| is not in an <a>excluded subtree</a>
+  * None of the ancestors of |C| up to |S| are in an <a>excluded subtree</a>
+
+Some elements are considered to be <dfn
+id="anchor-priority-candidates">priority candidates</dfn> for anchor selection:
+  1. An element that <a href="https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus">has focus</a>.
+  2. An element containing the <a href="https://drafts.csswg.org/css-ui-3/#insertion-caret">insertion caret</a>.
+  3. An element containing the current active selected match of the
+      find-in-page user-agent algorithm. If the match spans multiple elements, then
+      consider only the first such element.
+
+Note that if the <a>priority candidate</a> is an <a>atomic inline</a> element,
```

Comment:
This doesn't really make sense, atomic inlines are just fine.

Reply:
I've updated the text with the corrections